toolset/setup/linux/webservers/resin/resin.sh

```diff
@@ -4,12 +4,10 @@ fw_depends java
 
 fw_installed resin && return 0
 
-RVER=4.0.53
+RVER=4.0.55
 RESIN=resin-$RVER
 RESIN_HOME=$IROOT/$RESIN
 
-sudo cp -r $JAVA_HOME/include $JAVA_HOME/jre/bin/
```

Comment:
I **believe** some frameworks (maybe legacy support) relied on stuff in `include` to be on the path, but that can be added (if actually needed) to the path setup of `Java` later.